forgerock-openbanking-uk-aspsp-rs/forgerock-openbanking-uk-aspsp-rs-gateway/forgerock-openbanking-uk-aspsp-rs-gateway-server/src/test/java/com/forgerock/openbanking/aspsp/rs/api/fundsconfirmations/FundsConfirmationApiControllerIT.java

```diff
@@ -0,0 +1,91 @@
+/**
+ * Copyright 2019 ForgeRock AS.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.forgerock.openbanking.aspsp.rs.api.fundsconfirmations;
+
+import com.forgerock.openbanking.am.services.AMResourceServerService;
+import com.forgerock.openbanking.common.conf.RSConfiguration;
+import com.forgerock.openbanking.common.services.store.RsStoreGateway;
+import com.forgerock.openbanking.constants.OIDCConstants;
+import com.forgerock.openbanking.constants.OpenBankingConstants;
+import com.forgerock.openbanking.integration.test.support.SpringSecForTest;
+import com.forgerock.openbanking.jwt.exceptions.InvalidTokenException;
+import com.forgerock.openbanking.jwt.services.CryptoApiClient;
+import com.forgerock.openbanking.model.OBRIRole;
+import com.nimbusds.jwt.SignedJWT;
+import kong.unirest.JacksonObjectMapper;
+import kong.unirest.Unirest;
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.boot.web.server.LocalServerPort;
+import org.springframework.test.context.junit4.SpringRunner;
+import uk.org.openbanking.datamodel.fund.OBFundsConfirmationConsent1;
+import uk.org.openbanking.datamodel.fund.OBFundsConfirmationConsentResponse1;
+
+import java.io.IOException;
+import java.text.ParseException;
+
+import static com.forgerock.openbanking.integration.test.support.JWT.jws;
+import static org.mockito.BDDMockito.given;
+
+// TODO: DRAFT to finished
```

Comment:
Is is perhaps too early to add these ITs if they're not ready and don't yet do anything?